### PR TITLE
Non-functional cleanups and pylint satisfaction.

### PR DESCRIPTION
**Pull request by benjamin-work**
Mostly docstrings, indentation, ...

### CODE REVIEW

**Comment by benjamin-work** [comment]
@ottonemo pls review again

**Comment by benjamin-work** [comment]
Superseded by #188

**Comment by benjamin-work** [comment]
@ottonemo pls review again

File: pylintrc [hunk 1]
```diff
@@ -154,7 +154,7 @@ no-space-check =
 
 # Maximum number of lines in a module
 # DEFAULT:  max-module-lines=1000
-max-module-lines=1500
+max-module-lines=2500
```
Comment (ottonemo) [inline pylintrc]: I'm against this change. We should be remembered that `net.py` is too big and eventually split it up.

Comment (benjamin-work) [reply pylintrc]: 👍

**Comment by benjamin-work** [comment]
@ottonemo undid change to pylintrc

**Pull request merged**